docker/base/requirements.txt

```diff
@@ -1,8 +1,11 @@
 crcmod==1.7
 future==0.17.1
+lxml==4.5.0
```

Comment:
I'm not sure if this is needed after adding the same deps to `src/requirements.txt`

Reply:
https://github.com/google/clusterfuzz/pull/1712

Reply:
awesome!

Reply:
These deps here are also deprecated and Python 2 only.